factory/.env

```diff
@@ -21,16 +21,16 @@ NODE_VERSION="${FACTORY_DEFAULT_NODE_VERSION}"
 FACTORY_VERSION='4.2.0'
 
 # Chrome versions: https://www.ubuntuupdates.org/package/google_chrome/stable/main/base/google-chrome-stable
-CHROME_VERSION='128.0.6613.119-1'
+CHROME_VERSION='128.0.6613.137-1'
 
 # Cypress versions: https://www.npmjs.com/package/cypress
 CYPRESS_VERSION='13.14.2'
 
 # Edge versions: https://packages.microsoft.com/repos/edge/pool/main/m/microsoft-edge-stable/
-EDGE_VERSION='128.0.2739.63-1'
+EDGE_VERSION='128.0.2739.67-1'
```

Comment:
can't find revision 79 for some reason in the stable pool. will try again next week during the release

Reply:
I'm seeing https://packages.microsoft.com/repos/edge/pool/main/m/microsoft-edge-stable/microsoft-edge-stable_128.0.2739.79-1_amd64.deb from 12-Sep-2024.

Reply:
It's not critical if Edge `128.0.2739.79-1` isn't fully available. GitHub runners are currently using `128.0.2739.67`.